### PR TITLE
Documentation changes [ci skip]

### PR DESCRIPTION
**Pull request by siruguri**
* Add note on new idiom for migration methods - discussed in #1585 
* Mostly, correct punctuation and spelling
* Standardize capitalizations/upcase e.g., mime to MIME, paperclip to Paperclip

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -468,6 +468,8 @@ end
 
 ### Schema Definition
 
+As noted above, you can use the examples below, or switch to the idiom in Rails 3.2+ which uses `def up` and `def down` instead of `def self.up` and `def self.down`, respectively.
+
```
Comment (tute) [inline README.md]: Rails <3.2 is end-of-lifed, and not supported any longer (https://github.com/thoughtbot/paperclip/blob/master/.travis.yml#L14). I would remove this comment and old snippets.

File: README.md [hunk 2]
```diff
@@ -705,11 +707,11 @@ to inherit from `Paperclip::Processor` (see `lib/paperclip/processor.rb`).
 For example, when `:styles` are specified for an image attachment, the
 thumbnail processor (see `lib/paperclip/thumbnail.rb`) is loaded without having
 to specify it as a `:processor` parameter to `has_attached_file`.  When any
-other processor is defined it must be called out in the `:processors`
+other processor is defined, it must be called out in the `:processors`
 parameter if it is to be applied to the attachment.  The thumbnail processor
-uses the imagemagick `convert` command to do the work of resizing image
+uses the Imagemagick `convert` command to do the work of resizing image
```
Comment (tute) [inline README.md]: `ImageMagic` is camelized: https://en.wikipedia.org/wiki/ImageMagick

**Comment by tute** [comment]
Left two little comments, everything looks excellent otherwise! :+1:

**Comment by siruguri** [comment]
Thanks,  will send corrections early next week.

Typed on a mobile device.

**Pull request closed without merging**